Codigo/MODULO_NPCs.bas

```diff
@@ -1829,7 +1829,7 @@ Public Function CanMove(counter As t_NpcCounters, flags As t_NPCFlags) As Boolea
 End Function
 
 Public Function CanAttack(counter As t_NpcCounters, flags As t_NPCFlags) As Boolean
-    CanAttack = flags.Inmovilizado + flags.Paralizado = 0 And counter.StunEndTime < GetTickCount()
+    CanAttack = flags.Paralizado = 0 And counter.StunEndTime < GetTickCount()
 End Function
 
 Public Function StunNPc(ByRef Counters As t_NpcCounters)
```

Comment:
function no retorna ningun valor, make it a `sub` ? o hacela devolver un resultado

Reply:
como que no, retorna CanAttack

Reply:
me refiero a `Public Function StunNPc(ByRef Counters As t_NpcCounters)`

Reply:
ah me quede bien los cambios correspondientes a este pr